db/re/item_db_usable.yml

```
@@ -62359,7 +62359,7 @@ Body:
     NoUse:
       Sitting: true
     Script: |
-      /* TODO pet 2019; */
+      /* TODO pet; */
```

```suggestion
      /* TODO pet 2019; */
```
Otherwise the important information may be lost.